src/createStore.js

```diff
@@ -62,6 +62,7 @@ export default function createStore(reducer, initialState) {
 
     return function unsubscribe() {
       var index = listeners.indexOf(listener);
+      if (index === -1)return;
```

Comment:
Please be consistent with the project style.
In this case I'd rather write it as

```js
if (index !== -1) {
  listeners.splice(index, 1);
}
```